homeassistant/components/switchbot/config_flow.py

```diff
@@ -59,7 +59,7 @@ async def _get_switchbots(self) -> dict:
 
         # Discover switchbots nearby.
         async with connect_lock:
```

Comment:
Do we still need the lock?

Reply:
Hi @MartinHjelmare,

The lock is to limit concurrent requests to devices, they can only handle 1 request at a time.

Reply:
There's a single lock here and there's also a single lock in the library. Are these locks used in different ways, or can we remove the lock here in the integration and only use the library lock?

Reply:
I'm going to retest without the lock. Bluepy had a lot of erroneous disconnects and I might be mistaken about the bots.

Reply:
Tested it without the locks. Bleak handles btle scanning really well...had issue with this when issuing command.

There's no need for the lock anymore.